### PR TITLE
[RLlib] [Test] Add a logstd term to marwil's loss func to enocourage exploration.

### PR DESCRIPTION
**Pull request by gjoliver**
## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: rllib/agents/marwil/marwil_tf_policy.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import logging
 import gym
 from typing import Optional, Dict
+import sys
```
Comment (sven1977) [inline rllib/agents/marwil/marwil_tf_policy.py:4]: Can we do the same in `marwil_torch_policy.py`?

File: rllib/agents/marwil/marwil.py [hunk 2]
```diff
@@ -67,6 +67,9 @@
     # Number of steps to read before learning starts.
     "learning_starts": 0,
 
+    # A coeff to encourage good std for exploration.
```
Comment (sven1977) [inline rllib/agents/marwil/marwil.py:70]: Nice!

File: release/rllib_tests/learning_tests/hard_learning_tests.yaml [hunk 3]
```diff
@@ -1,470 +1,113 @@
...
-#            fcnet_activation: relu
-#            fcnet_hiddens: [256, 256, 256]
-
-#        evaluation_num_workers: 1
-#        evaluation_interval: 3
-#        evaluation_config:
-#            input: sampler
-
-cql-halfcheetahbulletenv-v0:
+ bc-halfcheetahbulletenv-v0:
```
Comment (sven1977) [inline release/rllib_tests/learning_tests/hard_learning_tests.yaml:1]: Not sure I understand these changes in the learning tests file. Let me know, if this was by mistake. :)

**Review by sven1977 — COMMENTED** [review]
Nice! Just a few comments/questions.

File: rllib/agents/marwil/marwil_tf_policy.py (hunk 1 above)
Comment (gjoliver) [reply rllib/agents/marwil/marwil_tf_policy.py:4]: done

**Review by gjoliver — COMMENTED** [review]
PTAL.
Sorry @sven1977, I wasn't sure I was ready to check this in at the time, so didn't clean up the PR at all.

**Review by sven1977 — APPROVED** [review]
Thanks a ton @gjoliver !
Could you take a last look at the failing tests? LINT and trainer compilation tests?
Thx! :)

**Pull request merged**